### DISCUSSION TITLE
Dynamic Filter 'Like' not working on Custom Query Data

### DISCUSSION THREAD

**Discussion by fadliasyhariStar** [Budibase/budibase#12509] [Help]
When i use dynamic fiter 'Like' on my data provider from a custom query it's working like a filter 'Starts with' instead. Is there any way to fix this?
thanks in advance.

![image](https://github.com/Budibase/budibase/assets/134992316/4ce64782-90e6-4616-ba3c-fdf70634e9c1)
![image](https://github.com/Budibase/budibase/assets/134992316/d2cfad40-dfcb-4113-b7f0-59e47706fe2d)
![image](https://github.com/Budibase/budibase/assets/134992316/9c5d05ea-9ba7-4eae-93ee-fea8630136dc)
![image](https://github.com/Budibase/budibase/assets/134992316/1cea8ca3-d574-4d07-ad15-093751365896)
![image](https://github.com/Budibase/budibase/assets/134992316/60b1df8e-374a-4ec9-be70-f744ed334755)

**Comment by ConorWebb96** [comment]
Hey @fadliasyhariStar,

I haven't been able to replicate?
<img width="1019" alt="Screenshot 2023-12-06 at 11 58 33" src="https://github.com/Budibase/budibase/assets/126772285/1bbd0cb3-e8a5-4ebd-98e9-dc57c81b98b9">
<img width="1400" alt="Screenshot 2023-12-06 at 11 58 40" src="https://github.com/Budibase/budibase/assets/126772285/bb58dcca-5c3f-49d6-ba11-e697a0f40557">

I'm on the latest version of Budibase and using an internal Budibase db. Can you let me know the following;

- What version of Budibase are you running?
- What data source you using e.g. internal Budibase, mysql, postgres etc.
- What internet browser you are using?
- Are you self-hosted or cloud?

I may need additional information but the answers to these questions would be a good start.

Thanks in advance.

**Reply by fadliasyhariStar** [reply]
Hey @ConorWebb96, thanks for responding.

- Version : 2.13.9.
- Data source : Postgres
- Browser : Chrome
- Self-Hosted

This is the supporting screen recording when I use the filter
![Screencast from 06-12-23 20_22_36](https://github.com/Budibase/budibase/assets/134992316/32f3b074-5b5a-4852-9c7b-87bc8d777826)

**Reply by ConorWebb96** [reply]
Hey @fadliasyhariStar,

I've tried with everything but your version of Budibase (I'm using the latest.) Still haven't been able to replicate this issue. Below i've attached a screenshot of my version of Budibase.
<img width="1306" alt="Screenshot 2023-12-07 at 09 23 39" src="https://github.com/Budibase/budibase/assets/126772285/b209171c-fd3c-4eec-ae07-d7365c01be9b">

Heres the gif showing it in action.
![20231207_092056](https://github.com/Budibase/budibase/assets/126772285/b9181305-3e43-45e3-8807-0c6d6d227478)

Can you provide a few things?
- A simple app export were you have replicated the issue.
- Will also like need a Postgres table schema dump.
